returnn/datasets/audio.py

```diff
@@ -402,6 +400,24 @@ def get_total_num_seqs(self, *, fast: bool = False) -> int:
         self._lazy_init()
         return len(self._data)
 
+    def get_data_dtype(self, key):
+        if key == "data":
+            return "float32"
+        elif key == "classes":
+            return "int32"
+        elif key == "raw":
+            return "string"
+        elif key == "orth":
+            return "uint8"
+        else:
+            raise ValueError(f"{self}: unknown data key: {key}")
+
+    def get_data_keys(self):
+        keys = ["classes", "orth", "raw"]
```

Comment:
I think if `self.targets is None`, we should also omit `classes` here.

Reply:
It is always available in the `DatasetSeq` though. hmm

Reply:
Ah, but empty.

Reply:
Yes it should neither be there as well.